### PR TITLE
SC: fix a disabled test case for value transfer

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- Fixed test case that was disabled due to sequential nonce check while adding multi-bridge implementation.
- Moved bridge tests to the `node/sc` directory.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [x] Lint and unit tests pass locally with my changes (`$ make test`)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- #79 

## Further comments

n/a

### CODE REVIEW

**Comment by jeonghonah** [comment]
@KimKyungup PTAL.

File: node/sc/bridge_test.go [hunk 1]
```diff
@@ -222,7 +221,7 @@ loop:
 // TestBridgeHandleValueTransferNonceAndBlockNumber checks the following:
 // - the bridge allows the handle value transfer with only serialized nonce.
 // - the bridge correctly stores and returns the block number.
-func _TestBridgeHandleValueTransferNonceAndBlockNumber(t *testing.T) {
+func TestBridgeHandleValueTransferNonceAndBlockNumber(t *testing.T) {
```
Comment (KimKyungup) [inline node/sc/bridge_test.go]: We need to modify the function name/comment. We don't need to test sequential nonce. :)

Comment (jeonghonah) [reply node/sc/bridge_test.go]: Fixed. PTAL.

File: node/sc/bridge_test.go [hunk 2]
```diff
@@ -218,11 +217,11 @@ loop:
 	t.Fatal("fail to check monotone increasing nonce", "lastNonce", expectedNonce)
 }
 
-// TODO-Klaytn-ServiceChain: current value transfer does not support sequential nonce.
 // TestBridgeHandleValueTransferNonceAndBlockNumber checks the following:
-// - the bridge allows the handle value transfer with only serialized nonce.
+// - the bridge allows the handle value transfer with an arbitrary nonce.
+// - the bridge keeps sequential  handle nonce for the recovery.
```
Comment (KimKyungup) [inline node/sc/bridge_test.go]:
```suggestion
// - the bridge keeps sequential handle nonce for the recovery.
```

Comment (jeonghonah) [reply node/sc/bridge_test.go]: Fixed.

**Pull request merged**